### DISCUSSION TITLE
i am downloading a LOT of books. is their a order in that they are downloaded?

### DISCUSSION THREAD

**Discussion by xenduro** [openaudible/openaudible#998] [Q&A] [answered]
i am downloading a LOT of books. is their a order in that they are downloaded?

around 6500.

**Comment by openaudible** [comment, marked as answer]
They are downloaded in the order added to the queue. But it isn’t obvious what the order is. If you want, you can cancel the jobs (select all, then action cancel jobs) and add just a few books to download… then add the rest. The ones you added first will be downloaded first.

**Comment by xenduro** [comment]
Tks that would have been my guess. Only 2500 of the 6600 left

 

Gonna need to move to a external HD 3 tb not enoght
